packages/get-platform/src/__tests__/getPlatform.test.ts

```diff
@@ -128,7 +128,7 @@ describe('getPlatformInternal', () => {
       expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
       expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(`
         prisma:warn Prisma failed to detect the libssl/openssl version to use, and may not work as expected. Defaulting to "openssl-1.1.x".
-        Please manually install OpenSSL via \`apt-get update -y && apt-get install -y openssl\` and try installing Prisma again. If you're running Prisma on Docker, you may also try to replace your base image with \`node:lts-slim\`, which already ships with OpenSSL installed.
+        Please manually install OpenSSL via \`apt-get update -y && apt-get install -y openssl\` and try installing Prisma again. If you're running Prisma on Docker, add this command to your Dockerfile, or switch to an image that already has OpenSSL installed.
```

Comment:
>add this command to your Dockerfile,
Where is the command 🤔 ?

Reply:
`apt-get update -y && apt-get install -y openssl` would be the command. It's shown in the immediately precedent paragraph

Reply:
... Not sure how I didn't see it 🙈

Reply:
(I also had to reread. Hope users are smarter than us)